napari/layers/utils/text_manager.py

```diff
@@ -121,24 +162,56 @@ def add(self, properties: dict, n_text: int):
         n_text : int
             The number of text elements to add
         """
-        if self._mode in (TextMode.PROPERTY, TextMode.FORMATTED):
-            new_text, _ = format_text_properties(
-                self._text_format_string, n_text=n_text, properties=properties
-            )
-            self.values = np.concatenate((self.values, new_text))
+        warnings.warn(
+            trans._(
+                'TextManager.add is deprecated. Use TextManager.fill instead.'
```

Comment:
Sorry if i missed it, but I can't find the definition for `TextManager.fill()`

Reply:
Good catch! That method used to exist, but has since been replaced by `apply`. I updated the string to point to that instead.

Reply:
Thanks for the quick fix!